client/src/main/java/org/spine3/change/Mismatch.java

```diff
@@ -147,30 +163,35 @@ public static ValueMismatch of(boolean expected, boolean actual, int version) {
      *
      * @param expected the value expected by a command, or {@code null} if the command expects not populated field
      * @param actual   the value actual in an entity, or {@code null} if the value is not set
-     * @param version  the current version of the entity
-     * @return info on the mismatch
+     * @param newValue the value from a command, which we wanted to set instead of {@code expected}
+     *@param version  the current version of the entity  @return info on the mismatch
      */
-    public static ValueMismatch of(@Nullable Message expected, @Nullable Message actual, int version) {
+    public static ValueMismatch of(@Nullable Message expected,
+                                   @Nullable Message actual,
+                                   @Nullable Message newValue,
+                                   int version) {
         final ValueMismatch.Builder builder = ValueMismatch.newBuilder();
         if (expected != null) {
-            builder.setExpected(AnyPacker.pack(expected));
+            builder.setExpectedPreviousValue(AnyPacker.pack(expected));
         }
         if (actual != null) {
-            builder.setActual(AnyPacker.pack(actual));
+            builder.setActualPreviousValue(AnyPacker.pack(actual));
+        }
+        if (newValue != null) {
```

Comment:
There is no test for this row. Maybe there should be `set_default_new_value_if_it_was_passed_as_null`.